### PR TITLE
Fix signing build sigcheck test

### PR DESCRIPTION
**Pull request by ankitkumarr**

### CODE REVIEW

**Comment by ankitkumarr** [comment]
Also need to merge this to `dev`

File: build/BuildSteps.cs [hunk 1]
```diff
@@ -339,6 +339,12 @@ public static void TestSignedArtifacts()
                 client.DownloadFile(Settings.SignInfo.SigcheckDownloadURL, sigcheckPath);
             }
 
+            // https://peter.hahndorf.eu/blog/post/2010/03/07/WorkAroundSysinternalsLicensePopups
+            // Can't use sigcheck without signing the License Agreement
+            Console.WriteLine("Signing EULA");
+            Console.WriteLine(Shell.GetOutput("reg.exe", "ADD HKCU\\Software\\Sysinternals /v EulaAccepted /t REG_DWORD /d 1 /f"));
+            Console.WriteLine(Shell.GetOutput("reg.exe", "ADD HKU\\.DEFAULT\\Software\\Sysinternals /v EulaAccepted /t REG_DWORD /d 1 /f"));
```
Comment (ankitkumarr) [inline build/BuildSteps.cs]: I could just do `reg.exe ADD "HKCU\Software\Sysinternals\Sigcheck" /v EulaAccepted /t REG_DWORD /d 1 /f`

But probably fine to accept for all Sysinternals tools.

Comment (soninaren) [reply build/BuildSteps.cs]: Would prefer to have an alternate way of accepting EULA then reg.exe. We can use this for now to unblock ourselves.

**Comment by ankitkumarr** [comment]
\azp run

**Pull request merged**